src/components/inherited-map/api.ts

```diff
@@ -14,7 +14,7 @@ export const fetchArea = (
   areaController = new AbortController();
 
   return fetch(
-    `${djangoBaseUrl}/api/stat/?center_point=${center[1]}+${center[0]}&scale=${zoom}`,
+    `${djangoBaseUrl}/api/stat/?center_point=${center[1]!}+${center[0]!}&scale=${zoom}`,
```

Comment:
`@typescript-eslint/restrict-template-expressions` does not like potential `undefined` here. We can't use Observable `[number, number]` for the coordinate bacuse mobx does not support tuples: https://github.com/mobxjs/mobx-state-tree/issues/805